code/modules/jobs/job_types/assistant.dm

```diff
@@ -31,6 +31,9 @@ Assistant
 /datum/outfit/job/assistant
 	name = "Assistant"
 	jobtype = /datum/job/assistant
+	head = /obj/item/clothing/head/helmet/knight
+	suit = /obj/item/clothing/suit/armor/riot/knight
+	backpack_contents = list(/obj/item/claymore/april_fools=1,)
```

Comment:
How did a lowly peasant afford any of this? smh, my immersion

Reply:
they hit a guard with a toolbox and looted it from them